bridge/cypress/integration/evaluation-heatmap.spec.ts

```diff
@@ -0,0 +1,107 @@
+  });
+  it('should show and hide tooltips', () => {
+    const tileId = 'ktb-heatmap-tile-182d10b8-b68d-49d4-86cd-5521352d7a42';
+    heatmap
+      .assertTooltipIsVisible(false)
+      .mouseOverOnTile(tileId)
+      .assertTooltipIsVisible(true)
+      .mouseLeaveOnTile(tileId)
+      .assertTooltipIsVisible(false);
+  });
+  it('should enumerate duplicate dates', () => {
+    heatmap.assertXAxisLabelExistsOnce('2022-02-08 12:56 (1)').assertXAxisLabelExistsOnce('2022-02-08 12:56 (2)');
+  });
+  it('should show all dates', () => {
+    heatmap.assertXAxisTickLength(10);
+  });
+  it('should not show secondary highlight if clicked evaluation has no other to compare', () => {
+    heatmap
+      .clickScore('ktb-heatmap-tile-25ab0f26-e6d8-48d5-a08f-08c8a136a688')
+      .assertPrimaryHighlight(1)
+      .assertSecondaryHighlight(0);
+  });
+  it('should reduce X elements on many evaluations', () => {
+    const labels = range(1, 44, 2).map((value) => `2022-02-01 03:46 (${value})`);
+    heatmap.interceptWithManyEvaluations().assertXAxisTickLength(22).assertXAxisTickLabels(labels);
+  });
+  it('should not show expand button if indicator results with score are less than 10', () => {
+    heatmap.interceptWith10Metrics().assertExpandExists(false);
+  });
+  it('should set tiles disabled/enabled via legend', () => {
+    heatmap
+      .clickLegendCircle('pass')
+      .assertTileColor('ktb-heatmap-tile-182d10b8-b68d-49d4-86cd-5521352d7a42', 'pass', true)
+      .clickLegendCircle('pass')
+      .assertTileColor('ktb-heatmap-tile-182d10b8-b68d-49d4-86cd-5521352d7a42', 'pass', false)
+      .clickLegendCircle('warning')
+      .assertTileColor('ktb-heatmap-tile-182d10b8-b68d-49d4-86cd-5521352d7a42', 'warning', true)
+      .clickLegendCircle('warning')
+      .assertTileColor('ktb-heatmap-tile-182d10b8-b68d-49d4-86cd-5521352d7a42', 'warning', false)
+      .clickLegendCircle('info')
+      .clickExpandButton()
+      .assertTileColor('ktb-heatmap-tile-e074893e-a7f9-4fa8-9e7e-898937a3d2b6', 'info', true)
+      .clickLegendCircle('info')
+      .assertTileColor('ktb-heatmap-tile-e074893e-a7f9-4fa8-9e7e-898937a3d2b6', 'info', false)
+      .clickLegendCircle('fail')
+      .assertTileColor('ktb-heatmap-tile-52b4b2c7-fa49-41f3-9b5c-b9aea2370bb4', 'fail', true)
+      .clickLegendCircle('fail')
+      .assertTileColor('ktb-heatmap-tile-52b4b2c7-fa49-41f3-9b5c-b9aea2370bb4', 'fail', false);
+  });
+  it('should not show tooltip if tile is disabled', () => {
+    heatmap
+      .clickLegendCircle('pass')
+      .clickScore('ktb-heatmap-tile-182d10b8-b68d-49d4-86cd-5521352d7a42')
+      .assertTooltipIsVisible(false)
+      .clickLegendCircle('warning')
+      .clickMetric('go_routines3a', 'ktb-heatmap-tile-182d10b8-b68d-49d4-86cd-5521352d7a42')
+      .assertTooltipIsVisible(false);
+  });
+  it('should show score/metric tooltip accordingly', () => {
+    heatmap
+      .clickScore('ktb-heatmap-tile-182d10b8-b68d-49d4-86cd-5521352d7a42')
```

Comment:
Just a minor thing but `ktb-heatmap-tile-`  is a static prefix that could be moved inside the function. It would also be easier to just call this function with the `keptnContext` alone